File: app/src/main/java/ua/com/todd/smsforwading/Sms.java
package ua.com.todd.smsforwading;

import com.j256.ormlite.field.DataType;
import com.j256.ormlite.field.DatabaseField;
import com.j256.ormlite.table.DatabaseTable;

@DatabaseTable(tableName = "messages")
public class Sms {
    public final static String SMS_FIELD_MESSAGE = "message";

    @DatabaseField(generatedId = true)
    private int Id;

    @DatabaseField(canBeNull = false, dataType = DataType.STRING, columnName = SMS_FIELD_MESSAGE)
    private String message;

    public Sms() {
    }

    public Sms(String message) {
        this.message = message;
    }

    public String getMessage() {
        return message;
    }

    @Override
    public String toString() {
        return message;
    }
}

File: app/src/main/kotlin/ua/com/todd/smsforwading/Sms.kt
package ua.com.todd.smsforwading

import com.j256.ormlite.field.DataType
import com.j256.ormlite.field.DatabaseField
import com.j256.ormlite.table.DatabaseTable

DatabaseTable(tableName = "messages")
public class Sms(val mes: String? = null) {

    class object {
        public val SMS_FIELD_MESSAGE: String = "message"
    }

    DatabaseField(generatedId = true)
    private val Id: Int = 0
    DatabaseField(canBeNull = false, dataType = DataType.STRING, columnName = SMS_FIELD_MESSAGE)
    private val message: String? = mes

    override fun toString(): String = message as String
}
